src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Row, Col, Strip } from "@canonical/react-components";
+
+export type Props = {
+  copyrightYear: number;
+  companyName: string;
+};
+
+const Footer: React.FC<Props> = ({ copyrightYear, companyName }) => {
```

Comment:
We tend to use the `function` syntax for components, so in this case:

```js
function Footer({ copyRightYear }) {
```

Reply:
Come to think of it, the copyright year also shouldn't be a prop. As this component is JS we can use `new Date().getFullYear()` instead so it is always correct.

Reply:
I removed copyrightYear variable as well :)